src/main/java/nz/cri/gns/NZSHM22/opensha/enumTreeBranches/NZSHM22_DeformationModel.java

```diff
@@ -109,9 +109,35 @@
 
     SBD_0_4_HKR_LR_30("Hikurangi, Kermadec to Louisville ridge, 30km - higher overall slip rates, aka Kermits revenge",
             "FaultModel SBD_0_2_HKR_LR_30 and the next three deprecated ones",
-            "dm_hk_tile_parameters_highkermsliprate_v2.csv");
+            "dm_hk_tile_parameters_highkermsliprate_v2.csv"),
+
+    SBD_0_2A_HKR_LR_30_CTP1("Hikurangi, Kermadec to Louisville ridge, 30km - Creeping Trench Perturbed v1",
```

Comment:
@voj I admit to being confused now about the meaning of 04, 03, 02, 02A etc

Reply:
So am I. Do you want to check with @chrisdicaprio before we merge this?

Reply:
02 is a geodetic model for the rates assuming a creeping trench, 02A is a modification to that to fix some issues with the boundary of the model. 03 is a locked trench model. 04 is a special high slip rate model that we are not likely to use anymore. _CTP[1-4] and _LTP[1-4] are the models with noise added (confusingly indicating locked or creeping trench, though 02A vs 03 should tell you that)